Gemfile

```diff
@@ -1,4 +0,0 @@
-source "https://rubygems.org"
```

Comment:
In fact, the Travis build fails without a Gemfile, so we *do* need this, but it should point to `gemspec`, not explicitly list the gems.